src/app/api/services/dot-router/dot-router.service.spec.ts

```diff
@@ -96,6 +96,20 @@ describe('DotRouterService', () => {
         expect(service.goToEditPage).toHaveBeenCalledWith({ url: '/about/us' });
     });
 
+    it('should go to Starter page', () => {
+        spyOn(service, 'goToStarter');
+        service.goToStarter();
+
+        expect(service.goToStarter).toHaveBeenCalledTimes(1);
+    });
+
+    it('should go to Content page', () => {
+        spyOn(service, 'goToContent');
+        service.goToContent();
+
+        expect(service.goToContent).toHaveBeenCalledTimes(1);
+    });
+
```

Comment:
This test is not right you are only testing that you called the method but not what happen inside the method, what you need to assert here is that when you call those methods the `this.router.navigate` was called with the specific url like this one: https://github.com/dotCMS/core-web/blob/9b20c831fda4c8d10e9083b3001f6325afce5fd5/src/app/api/services/dot-router/dot-router.service.spec.ts#L113